server/fileblame.go

```diff
@@ -462,6 +469,57 @@ func extendDiff(
 	return lines, content_lines, nil
 }
 
+var logPaginationLimit = 3
+
+func buildLogData(
+	repo config.RepoConfig,
+	gitHistory *blameworthy.GitHistory,
+	path string,
+	offset int,
+	data *LogData,
```

Comment:
As I'm still learning Go: what's the advantage to passing this in instead of creating it here and returning it?

Reply:
None, I did this to follow the convention used in this file. I'll change it.